olio/dao/connection_manager.go

```diff
@@ -54,7 +54,10 @@ func NewConnectionManager() *ConnectionManager {
 	connectionManager := ConnectionManager{}
 
 	dbConnectionString := util.GetEnv("DB_CONNECTION_STRING", "root:root@/todo?parseTime=true")
-	dialect := util.GetEnv("DB_CONNECTION_DIALECT", "mysql")
+	dialect, err := util.DbDialect(dbConnectionString)
+	if err != nil {
+		panic(err.Error())
```

Comment:
maybe this should be more explicit, like
"failed to parse db dialect from connection string: " + err.Error()